auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -460,6 +465,9 @@ private boolean makeCheckRightCall(String actorCrn, String userCrn, String right
     public List<Boolean> hasRightsNoCache(String actorCrn, String memberCrn, List<RightCheck> rightChecks, Optional<String> requestId) {
         LOGGER.info("Checking whether member [{}] has rights [{}]", memberCrn,
                 rightChecks.stream().map(this::rightCheckToString).collect(Collectors.toList()));
+        checkArgument(rightChecks.stream().map(RightCheck::getResource).allMatch(Crn::isCrn),
```

Comment:
The resource can be `null` or '*' as well, which case UMS does an account level check, so I would use the condition `c -> c == null || "*".equals(c) || Crn.isCrn(c)`.

Reply:
null is not allowed on the interface, so it does not work anyway, i am suprised that '*' is allowed, since it is not a check for a specific right, but i'll fix this, but I think we will never send null or '*' on this interface

Reply:
fixed